### ISSUE TITLE
Team update - Sep 28, 2022

### ISSUE THREAD

**Issue by trallard** [Quansight-Labs/jupyter-a11y-mgmt#160]
This is a @Quansight-Labs/czi-a11y-grant team sync 🎉🎉🎉! This is a way for the Team Members to provide status reports on what they've been up to this week and request help and attention for things they are working on. This issue will be closed at the end of the day.

_Copy and paste the template below, and answer questions as you wish!_

<details>
<summary>Response Template</summary>

```md
**Thanks I'd like to give 🙌**
- So-and-so helped me out a lot with foo...

**Updates from last week :heavy_check_mark:**
- I worked towards deliverable: <link-to-deliverable>

**Challenges I faced and things I'd like assistance with 🙏**
- I had a hard time figuring out ...

**My availability for next week**
- I'll be off on foo day...

**Important items for discussion 💬**
- I have a question about goal <link-to-goal>
- Can @foo give a comment on issue #NN ?
- I opened #NN for discussion, please chime in
- I need to discuss foo
```

</details>

#### :mag: Needs Triage

_The [Needs Triage issues](https://github.com/Quansight-Labs/jupyter-a11y-mgmt/labels/status%3A%20needs%20triage) require an initial assessment and labeling._

No issues need triage! 🎉

### :dart: Project boards

Please make sure the boards reflect the current status of the project.

[Jupyter a11y grant project](https://github.com/orgs/Quansight-Labs/projects/5/views/5)

**Comment by steff456** [comment]
**Updates from last week :heavy_check_mark:**
- I worked in the new pygments package: https://github.com/Quansight-Labs/accessible-pygments
- Created an issue for adding the hamburger menu in Jupyterlab: https://github.com/jupyterlab/jupyterlab/issues/13148

**Challenges I faced and things I'd like assistance with 🙏**
- I'm ok

**My availability for next week**
- I'll be here

**Important items for discussion 💬**
- I don't have anything super important to discuss

**Comment by isabela-pf** [comment]
**Thanks I'd like to give 🙌**

**Updates from last week :heavy_check_mark:**
- #133 has [new mockups for 200% zoom behavior](https://github.com/Quansight-Labs/jupyterlab-accessible-themes/issues/34#issuecomment-1262458172).
- #146 planning for next set of tests. I have a timeline for the end of the year and polishing the things that came from the last test into PR form.

**Challenges I faced and things I'd like assistance with 🙏**
- I think I'm okay right now.

**My availability for next week**
- I'll be here all week. 

**Important items for discussion 💬**
- Eventually the new mockups, but I think that doesn't fit as well in an update-focused meeting.

**Comment by trallard** [comment]
- @steff456
  - working on pygments - some pygments working there already 🎉  needs some polishing and bits here and there
  - will then move onto the hamburger/ellipsis fix issues

---

- @gabalafou 
  - will be moving to Colombia next week - so timezones will be slightly different from usual
  - 🎉 got his `expect-axe-playwright` PR merged after a while
  - learned about `pa11y` + CI  

---

- @kulsoomzahra 
  - working on https://github.com/Quansight-Labs/jupyterlab-accessible-themes/issues/37
  - preparing for show and tell on Monday

--

- @isabela-pf 
  -  new suggestions for UI https://github.com/Quansight-Labs/jupyterlab-accessible-themes/issues/34#issuecomment-1262458172
  - Space telescope - planning new user testing at the end of the year (content focused)

**Comment by gabalafou** [comment]
**Thanks I'd like to give 🙌**
- @isabela-pf for organizing the vote to elect a representative from the Jupyter Accessibility Subproject to the Jupyter Software Steering Council.

**Updates from last week :heavy_check_mark:**

I made a mistake this week and thought, maybe I'll use Monday and Tuesday to clear some small tasks from my plate, and then those ended up just taking up all my time. So the following bullet points are things I've worked on that are accessibility-related but not directly related to accessibility in Jupyter.

- Good news: Finished upstreaming my changes to `expect-axe-playwright`. Bad news: not sure we'll use it, and it took too much of my time this week.
- https://github.com/Widen/expect-axe-playwright/pull/24. Finished this PR, which felt like doing a little bit of public service. The aim here was to prevent unintentionally spreading the idea among developers that there's an automated tool out there that can stamp your page or app as "accessible" or not.
- https://github.com/Widen/expect-axe-playwright/pull/23. Finished this PR. It will allow us to stop using my fork of expect-axe-playwright. 
- https://github.com/Quansight/Quansight-website/pull/262. Worked some more on adding Pa11y-CI to the Quansight website. Discovered more obstacles, specifically the default configuration for Axe flags a bunch of elements on the site for contrast that might not have contrast issues. These elements either have a background image or otherwise overlap some other element, and for some reason, that makes Axe incapable of ascertaining the foreground/background contrast ratio. So I'm going to have to hide a bunch of elements during testing in order to get the site to pass the CI/CD run or figure out a way to configure Axe differently.

**My availability for next week**
- I am taking Monday off.
